app/src/main/java/de/danoeh/antennapod/adapter/DownloadLogAdapter.java

```diff
@@ -165,4 +174,16 @@ public long getItemId(int position) {
         DownloadStatus getItem(int position);
     }
 
+    public void retryAll() {
+        for (int i = 0; i < getCount(); i++) {
+            DownloadStatus status = getItem(i);
+            if (status != null && status.getFeedfileType() == FeedMedia.FEEDFILETYPE_FEEDMEDIA) {
```

Comment:
This will also retry items that were previously unsuccessful and then successful afterwards. I think this should only retry items with `newerWasSuccessful(...)`